docassemble/AssemblyLine/data/questions/interview_list.yml

```diff
@@ -30,53 +79,89 @@ modules:
+    CourtFormsOnline
     % endif
   short title: |
-    % if get_config("interview page title"):
-    ${ get_config("interview page title") }
-    % else:    
+    % if interview_page_title:
+    ${ interview_page_title }
+    % elif showifdef("AL_ORGANIZATION_TITLE"):
     ${ AL_ORGANIZATION_TITLE }
-    % endif 
+    % else:
+    CourtFormsOnline
+    % endif
   title url: |
-    ${ all_variables(special='metadata').get('title url', AL_ORGANIZATION_HOMEPAGE) }
+    % if interview_page_url:
+    ${ interview_page_url }
+    % elif showifdef("AL_ORGANIZATION_HOMEPAGE"):
+    ${ AL_ORGANIZATION_HOMEPAGE }
+    % else:
+    % endif:
   exit url: |
-    ${ all_variables(special='metadata').get('exit url', AL_ORGANIZATION_HOMEPAGE) }
+    % if interview_page_url:
+    ${ interview_page_url }
+    % elif showifdef("AL_ORGANIZATION_HOMEPAGE"):
+    ${ AL_ORGANIZATION_HOMEPAGE }
+    % else:
+    % endif:
   logo: |
     <span class="title-container">
       <span class="al-logo">
+    % if logo_url:
+        <img src="${ logo_url }" alt="${ logo_alt }"/>
+    % else:
         <img src="${ al_logo.url_for() }" alt="${ al_logo.alt_text }"/>
+    % endif
       </span>
       <span class="al-title">
-        <span class="title-row-1">${ AL_ORGANIZATION_TITLE }</span>
-        <span class="title-row-2">${all_variables(special='metadata').get('title','').rstrip()}</span>
+    % if interview_page_title:
+        <span class="title-row-1">${ interview_page_title }</span>
+    % elif showifdef("AL_ORGANIZATION_TITLE"):
+        <span class="title-row-1">${ AL_ORGANIZATION_TITLE }</span>    
+    % else:
+        <span class="title-row-1">CourtFormsOnline</span>    
+    % endif
+    % if interview_page_heading:
+        <span class="title-row-2">${ interview_page_heading }</span>
+    % else:
+        <span class="title-row-2">${ all_variables(special='metadata').get('title','').rstrip() }</span>
+    % endif
       </span>
     </span>
   short logo: |
     <span class="title-container">
       <span class="al-logo">
+    % if logo_url:
+        <img src="${ logo_url }" alt="${ logo_alt }"/>
+    % else:
         <img src="${ al_logo.url_for() }" alt="${ al_logo.alt_text }"/>
+    % endif
       </span>
       <span class="al-title">
-        <span class="title-row-1">${ AL_ORGANIZATION_TITLE }</span>
-        <span class="title-row-2">${ all_variables(special='metadata').get('short title','').rstrip() }</span>
+    % if interview_page_title:
+        <span class="title-row-1">${ interview_page_title }</span>
+    % elif showifdef("AL_ORGANIZATION_TITLE"):
+        <span class="title-row-1">${ AL_ORGANIZATION_TITLE }</span>    
+    % else:
+        <span class="title-row-1">CourtFormsOnline</span>    
+    % endif
+    % if interview_page_heading:
+        <span class="title-row-2">${ interview_page_heading }</span>
+    % else:
+        <span class="title-row-2">${ all_variables(special='metadata').get('title','').rstrip() }</span>
+    % endif
```

Comment:
Looking at this again, I almost wish there were a `logo_title_row1` and a `...row2` - variables that would be independent of `interview_page_title` and `interview_page_heading/question`. I'm not sure I like those doing double duty and I could see why someone might want them to be different.

Reply:
I guess I can make that separately configurable.